frontend/public/components/utils/webhooks.tsx

```diff
@@ -16,22 +23,32 @@ enum TriggerTypes {
 const webhookTriggers = new Set<TriggerTypes>([TriggerTypes.Bitbucket, TriggerTypes.Generic, TriggerTypes.GitHub, TriggerTypes.GitLab]);
 const getTriggerProperty = trigger => trigger.type.toLowerCase();
 
-export const WebhookTriggers: React.SFC<WebhookTriggersProps> = ({ resource }) => {
+const getTableColumnClasses = canGetSecret => {
+  if (canGetSecret) {
+    return [classNames('col-lg-2', 'col-md-4', 'col-sm-4', 'col-xs-6'),
```

Comment:
style nit: newlines after `[` and before `]`. Otherwise it's really easy to miss that this is an array.